### PR TITLE
[Ansible] Add `restart-ptf` to `testbed-cli.sh`

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Add `restart-ptf` to restart ptf docker container by adding new function
`restart_ptf` to `testbed-cli.sh`, it basically does the same thing as
`renumber_topo` without setting out fanout switch.

Also fix some bugs of `vm_topology.py` and `renumber_topo.yml`:
1. Add back plane port when renumbering topology.
2. Call `create_dut_port.yml` to setup vlan virtual interfaces before
`vm_topology`.
3. Add needed steps for other testbeds like `fullmesh`.

Signed-off-by: Longxiang Lyu <lolv@microsoft.com>

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
Support restarting ptf docker container.

#### How did you do it?
Add `restart-ptf` to restart ptf docker container by adding new function
`restart_ptf` to `testbed-cli.sh`, it basically does the same thing as
`renumber_topo` without setting out fanout switch.

Also fix some bugs of `vm_topology.py` and `renumber_topo.yml`:
1. Add back plane port when renumbering topology.
2. Call `create_dut_port.yml` to setup vlan virtual interfaces before
`vm_topology`.
3. Add needed steps for other testbeds like `fullmesh`.

#### How did you verify/test it?
* tested it over t0, dualtor.
```
./testbed-cli.sh restart-ptf vms20-t0-7050cx3-2 password.txt
./testbed-cli.sh restart-ptf vms21-dual-t0-7050-3 password.txt
```

### CODE REVIEW

File: ansible/ansible.cfg [hunk 1]
```diff
@@ -26,6 +26,9 @@ transport      = smart
 module_lang    = C
 max_diff_size  = 512000
 
+# enable stdout plugin to debug
+stdout_callback = debug
+
```
Comment (yxieca) [inline ansible/ansible.cfg:31]: Please make this change in a separate PR. Thanks

Comment (lolyu) [reply ansible/ansible.cfg]: updated, thanks.

**Comment by yxieca** [comment]
@lolyu please look into the LGTM warnings.

**Comment by yxieca** [comment]
> @lolyu please look into the LGTM warnings.

Never mind, these are fixed warnings.

**Review by yxieca — APPROVED** [review]
